project/src/test/java/com/google/sps/EventSpannerTasksTest.java

```diff
@@ -116,36 +128,68 @@ public void getEventsByIdTest() {
 
   /**
    * Verify putting event in database through /create-event doPost with correct params and
-   * getting back correct event out of db
+   * getting back correct event out of db without going through NLP API
    */
   @Test
-  public void testEventCreationDoPost() throws Exception {
+  public void testEventCreationDoPost_noNlp() throws Exception {
     SpannerTasks.insertOrUpdateUser(HOST);
-    request.addParameter("name", EVENT_NAME);
-    request.addParameter("date", DATE_STRING);
-    request.addParameter("time", TIME);
-    request.addParameter("description", DESCRIPTION);
-    request.addParameter("location", LOCATION);
-    authenticationHelper
-        .setEnvIsLoggedIn(true)
-        .setEnvEmail(EMAIL)
-        .setEnvAuthDomain("example.com");
+    setAuthenticationHelper();
+    Mockito.when(request.getParameter("name")).thenReturn(EVENT_NAME);
+    Mockito.when(request.getParameter("date")).thenReturn(DATE_STRING);
+    Mockito.when(request.getParameter("time")).thenReturn(TIME);
+    Mockito.when(request.getParameter("description")).thenReturn(DESCRIPTION);
+    Mockito.when(request.getParameter("location")).thenReturn(LOCATION);
+    Mockito.when(request.getParameter("interests")).thenReturn("Tech, Work");
 
     new EventCreationServlet().doPost(request, response);
+    Event event =
+        new Event.Builder(EVENT_NAME, DESCRIPTION, LABELS, LOCATION, DATE, TIME, HOST).setId(stringWriter.toString().trim().split("\"")[3]).build();
```

Comment:
This line seems a little bit long to me. Can you check if it is shorter than 80?

Reply:
done, thanks